src/main/java/net/javadiscord/javabot/util/IndentationHelper.java

```diff
@@ -0,0 +1,179 @@
+package net.javadiscord.javabot.util;
```

Comment:
Please use tabs for formatting your code. See [the checkstyle failure](https://github.com/Java-Discord/JavaBot/actions/runs/4708493112/jobs/8351238910?pr=407).